pkg/kubelet/cm/memorymanager/state/state_mem.go

```diff
@@ -94,6 +94,7 @@ func (s *stateMemory) SetMemoryAssignments(assignments ContainerMemoryAssignment
 	defer s.Unlock()
 
 	s.assignments = assignments.Clone()
+	klog.InfoS("Updated Memory assignments", "assignments", assignments)
```

Comment:
assigments can be large, because of this I'd do this with `V(5)` at least

Reply:
Ack, will update.